package.json

```diff
@@ -33,5 +33,10 @@
   },
   "engines": {
     "node": ">=14"
+  },
+  "dependencies": {
```

Comment:
i think we should/could move this to dev dependencies, as we don't include those dependencies in the build result. @PhilippSiegmantel what do you think?

Reply:
I agree with that. It's probably already a peer dep, because Grafana provides these on its own.